app/templating/template_renderer.py

```diff
@@ -5,52 +5,73 @@
 import re
 from jinja2 import Environment
 
-from app.jinja_filters import format_date, format_household_member_name, format_currency, format_number,\
-    get_currency_symbol, format_household_summary, format_conditional_date, format_unordered_list, \
-    format_date_range, format_household_member_name_possessive, concatenated_list, calculate_years_difference, \
-    get_current_date
+import app.jinja_filters as filters
 
 
 class TemplateRenderer:
     def __init__(self):
-        self.environment = Environment()
-
-        self.environment.filters['format_date'] = format_date
-        self.environment.filters['format_household_name'] = format_household_member_name
-        self.environment.filters['format_household_name_possessive'] = format_household_member_name_possessive
-        self.environment.filters['format_household_summary'] = format_household_summary
-        self.environment.filters['format_unordered_list'] = format_unordered_list
-        self.environment.globals['format_conditional_date'] = format_conditional_date
-        self.environment.filters['format_currency'] = format_currency
-        self.environment.filters['format_number'] = format_number
-        self.environment.filters['get_currency_symbol'] = get_currency_symbol
-        self.environment.globals['format_date_range'] = format_date_range
-        self.environment.filters['concatenated_list'] = concatenated_list
-        self.environment.globals['calculate_years_difference'] = calculate_years_difference
-        self.environment.globals['get_current_date'] = get_current_date
+        env = Environment()
+        env.filters['format_date'
```

Comment:
Ick - I may need to argue with the linter over this. Splitting square brackets looks really odd to me.

Ideally I'd say this is a good example of where the 80 char limit makes things unnecessarily difficult.

Possible alternative:
```
env.filters['format_household_name_possessive'] = (
    filters.format_household_member_name_possessive
)
```

Or a bit more left-of-field:

```
included_filters = ['format_date', 'format_month_year_date', 'etc...']
for name in included_filters:
  env.filters[name] = getattr(filters, name)

included_globals = ['format_conditional_date', 'format_date_range', 'etc...']
for name in included_globals:
  env.globals[name] = getattr(filters, name)
```

Reply:
Agree it looks odd.  I very much like the latter (left-of-field) - thanks that's a good steer ;)